### ISSUE TITLE
Ender IO - 1.12.2 - 5.0.29 Fatal Recipe Error

### ISSUE THREAD

**Issue by xCARVENx** [SleepyTrousers/EnderIO#4818]
#### Issue Description:
The error itself is simple, but all the changes that I have made to correct this error without result. Put me on the right path.
Added recipes for creative things in the folder scripts. After that, as I understand, there was an error in the recipes of the gradient ingot with wooden gears. 
### =============================
== ENDER IO FATAL RECIPE ERROR ==
### =============================
Cannot register recipes as configured. This means that either your custom recipe files 
have an error or another mod does bad things to vanilla items or the Ore Dictionary. 
### =============================
== Bad file ==
 recipes/materials.xml
### =============================
 == Error Message ==
Could not find a crafting ingredient for 'dustCoal in <recipe> 'Capacitor 2'
### ===========================
Note: If this is a modpack, report to the modpack author, not to the Ender IO team.
### =============================
 A custom exception was thrown by a mod, the game will display an error screen and halt.
com.enderio.core.client.EnderCoreModConflictException: 
#### What you expected to happen:
At the end of the game loading produces such an error. I tried to change the ingredients of the craft, changed the name, disconnected the craft - but all without result.

#### Steps to reproduce:

1.Install Forge 14.23.4.2729 / Ender IO - 1.12.2-5.0.29 / Ender Core - 1.12.2-0.5.35
2.to throw recipes in scripts
3.Load up the game
...

____
#### Affected Versions:

- EnderIO:1.12.2-5.0.29
- EnderCore:1.12.2-0.5.35
- Minecraft:1.12.2
- Forge:14.23.4.2729
- SpongeForge: no
- Optifine: no
- Single Player

#### Your most recent log file where the issue was present: 

[pastebin/gist/etc link here]
[latest.log](https://github.com/SleepyTrousers/EnderIO/files/2259483/latest.log)
folder with scripts: https://yadi.sk/d/8Fkm2mEo3Ztd65
Latest.log: https://pastebin.com/f6JURqd7

**Comment by tyler489** [comment]
Have you disabled wooden gears or removed them from an oredict

**Comment by xCARVENx** [comment]
I need to include them in: config/enderio/materials.xml ?

**Comment by xCARVENx** [comment]
Here like would find in: minecraft/scripts/oredict/buildcraft.zs
[buildcraft.txt](https://github.com/SleepyTrousers/EnderIO/files/2259529/buildcraft.txt)

**Comment by tyler489** [comment]
Whats the buildcraft id of there gear.

**Comment by HenryLoenwind** [comment]
If you were using 5.0.30, you could just replace the alias in a user recipe.

But as is, I'd recommend to not mess with oredicts. Those not configurable like recipes for a reason.

**Comment by xCARVENx** [comment]
Now I'll try, I noticed that in oredict use the alias gear_wood, and in the enderio / materials GEAR_WOOD. It can somehow influence.

**Comment by xCARVENx** [comment]
It looks like I'm on the right track, now another mistake. Probably the same in the difference of names.
[latest.log](https://github.com/SleepyTrousers/EnderIO/files/2259549/latest.log)

**Comment by tyler489** [comment]
If u could put that on something like pastebin i can try to help.. i am unable to read .log files on my phone

**Comment by xCARVENx** [comment]
Sorry for the misunderstanding now corrected

**Comment by xCARVENx** [comment]
https://pastebin.com/f6JURqd7

**Comment by tyler489** [comment]
You removed dustCoal oredict too?

**Comment by xCARVENx** [comment]
No, after fixing the wooden gear, a problem arose. Now I'm trying to figure out where to edit.

**Comment by xCARVENx** [comment]
I understand the whole point in the scripts of the recipes of creative things, they use wrong names. I did not disconnect all these things.

**Comment by xCARVENx** [comment]
I managed to start and create the world, but the crafts from the folder scripts are missing.

**Comment by xCARVENx** [comment]
On the previous issue, I replaced the incorrect name dustCoal in: scripts/unify/baze.zs and scripts/mod_integrations/immersive_engineering.zs , the correct POWDER_COAL in: config/enderio/recipes/aliases.xml

**Comment by xCARVENx** [comment]
I found a mistake! Maybe it was for someone clear, but not someone who has not laid out detailed instructions for solving this problem, so I will count it on my own account.
For some reason, in JEI configurations а lot of prohibited items!
[itemBlacklist.txt](https://github.com/SleepyTrousers/EnderIO/files/2298338/itemBlacklist.txt)
A lot of prohibited items!
I did not change anything, he automatically brought a bunch of items to the black list, I do not know how it happened. I just cleaned the list, that's all))
This is all strange and the solution lay on the surface.

But anyway, thanks to all at least for not having shut down and ignoring my problem.)
